blocks/editable/content.js

```diff
@@ -0,0 +1,117 @@
+import { renderToString } from '@wordpress/element';
+import { last } from 'lodash';
+import { childrenToReact } from './tree';
+
+export function nodeListToReact( editor, nodeList ) {
+	const fragment = editor.getDoc().createDocumentFragment();
+
+	nodeList.forEach( function( node ) {
+		fragment.appendChild( node.cloneNode( true ) );
+	} );
+
+	return childrenToReact( editor.serializer.serialize( fragment, { format: 'tree' } ) );
+}
+
+function splitResult( before, after ) {
+	return { before: before, after: after };
+}
+
+export function setContent( editor, content ) {
+	if ( ! content ) {
+		content = '';
+	}
+
+	content = renderToString( content );
+	editor.setContent( content, { format: 'raw' } );
```

Comment:
Can we use the "tree" format to set content as well?

Reply:
Currently we only support getting the contents out as the tree but since it's an internal thing we just need to expose I see no reason why we couldn't support setting contents as a tree as well that would by pass the parsing step in the setContent call so it would be more efficient obviously.

Reply:
That would be awesome, if we could send the same tree back 💯